app/elements/devguide-shell.html

```diff
@@ -216,17 +269,42 @@
     },
 
     attached: function() {
+      Polymer.RenderStatus.afterNextRender(this, function() {
+        // Somebody might look at the API docs. Be ready.
+        this.importHref(this.resolveUrl('/elements/iron-doc-viewer.html'));
+      });
+
       // Compute the active page from the parent pw-shell's app route.
       var pwShell = this.parentElement;
       if (pwShell) {
         var path = pwShell._sectionRoute.path;
+
         // If it ends in a /, then it's a top level path like /1.0/tools/,
         // so add an index to it.
         if (path[path.length - 1] === '/') {
           path += 'index';
         }
         this.activePage = path;
       }
+
+      // Figure out which version we're displaying, 1.0 or 2.0. You can't rely on
+      // the parent's version, because if we clicked on the "see 2.0" <a> link inside
+      // this page, then the pw-shell would not have refreshed, so it doesn't
+      // know the version has changed.
+      pwShell.updateDocsVersion();
```

Comment:
If pw-shell gets version from URL, calling this method shouldn't be needed.

Since a new instance of devguide-shell is being created on every page nav (remember, ALL the children of pw-shell is replaced), there's no need to keep `<devguide-shell>.activeDocsVersion` updated with `_setActiveDocsVersion`. Instead, the server template can set the current version through HTML attributes:

base-devguide.html:
```html
<devguide-shell v1-page-link="{{{versions[0]}}}" v2-page-link="{{{versions[1]}}}" active-docs-version="{{{version}}}">
```